xml/System.Collections.Generic/HashSet`1.xml

```diff
@@ -127,69 +127,15 @@
-  
- The following table shows the <xref:System.Collections.Generic.HashSet%601> operations and their equivalent LINQ set operations.  
-  
-|HashSet operation|LINQ equivalent|  
-|-------------------------------|---------------------|  
-|<xref:System.Collections.Generic.HashSet%601.UnionWith%2A>|<xref:System.Linq.Enumerable.Union%2A>|  
-|<xref:System.Collections.Generic.HashSet%601.IntersectWith%2A>|<xref:System.Linq.Enumerable.Intersect%2A>|  
-|<xref:System.Collections.Generic.HashSet%601.ExceptWith%2A>|<xref:System.Linq.Enumerable.Except%2A>|  
-|Not provided.|<xref:System.Linq.Enumerable.Distinct%2A>|  
-|<xref:System.Collections.Generic.HashSet%601.SymmetricExceptWith%2A>|Not provided.|  
-|<xref:System.Collections.Generic.HashSet%601.Overlaps%2A>|Not provided.|  
-|<xref:System.Collections.Generic.HashSet%601.IsSubsetOf%2A>|Not provided.|  
-|<xref:System.Collections.Generic.HashSet%601.IsProperSubsetOf%2A>|Not provided.|  
-|<xref:System.Collections.Generic.HashSet%601.IsSupersetOf%2A>|Not provided.|  
-|<xref:System.Collections.Generic.HashSet%601.IsProperSupersetOf%2A>|Not provided.|  
-|<xref:System.Collections.Generic.HashSet%601.SetEquals%2A>|Not provided.|  
-  
-   
-  
-## Examples  
- The following example demonstrates how to merge two disparate sets. This example creates two <xref:System.Collections.Generic.HashSet%601> objects, and populates them with even and odd numbers, respectively. A third <xref:System.Collections.Generic.HashSet%601> object is created from the set that contains the even numbers. The example then calls the <xref:System.Collections.Generic.HashSet%601.UnionWith%2A> method, which adds the odd number set to the third set.  
-  
+    <remarks>For more information about this API, see <see href="/dotnet/fundamentals/runtime-libraries/system-collections-generic-hashset{t}">Supplemental API remarks for HashSet&lt;T&gt;</see>.</remarks>
+    <example>
+      <format type="text/markdown"><![CDATA[
+The following example demonstrates how to merge two disparate sets. This example creates two <xref:System.Collections.Generic.HashSet%601> objects and populates them with even and odd numbers, respectively. A third <xref:System.Collections.Generic.HashSet%601> object is created from the set that contains the even numbers. The example then calls the <xref:System.Collections.Generic.HashSet%601.UnionWith%2A> method, which adds the odd number set to the third set.
+
  :::code language="csharp" source="~/snippets/csharp/System.Collections.Generic/HashSetT/Overview/Program.cs" interactive="try-dotnet-method" id="Snippet01":::
- :::code language="vb" source="~/snippets/visualbasic/VS_Snippets_CLR_System/system.Collections.Generic.HashSet_UnionWith/vb/Program.vb" id="Snippet01":::  
-  
- ]]></format>
-    </remarks>
+ :::code language="vb" source="~/snippets/visualbasic/VS_Snippets_CLR_System/system.Collections.Generic.HashSet_UnionWith/vb/Program.vb" id="Snippet01":::
```

Comment:
I suppose these interactive examples don't have an equivalent xml element, so they'll have to stay as CDATA.

We'll have to discuss what to do with them when we backport them, as they are interactive in this repo.

Reply:
Can't you just backport them as CDATA?